subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/Result.kt

```diff
@@ -0,0 +1,46 @@
+package com.avito.runner.service.worker.device.adb
```

Comment:
prefer to see that in a separated module

Reply:
I decided to extract Result type and replace funktionale in all modules to it in separate PR